### DISCUSSION TITLE
"Move" existing installs to scoop/apps?

### DISCUSSION THREAD

**Discussion by boardtc** [ScoopInstaller/Scoop#5741] [General]
I had a bad experience with _WinGetUI_ this week and tried _Chocolatey_ in the past but wasn't into it. I heard about _Scoop_ today and gave it a go to install OpenLens.

From the [quick start page](https://scoop.sh/), I did a search and then following the message: _scoop bucket add extras_. It would be worth adding a line in the search panel to show you then run the command scoop install mongo...

Anyway, _scoop install openlens_, installed 7-zip under the scoop/apps directory. But 7-zip is already on my machine. I don't want 2 copies.

It looks like _Scoop_ does not look beyond it's sandbox. Has a script been considered to "move" apps from wherever they are installed to _Scoop_?

**Comment by HUMORCE** [comment]
For 7-zip: `scoop config 7ZIPEXTRACT_USE_EXTERNAL true`

For other apps: There is no way, Scoop cannot convert them to Scooped apps.
